web/libs/editor/src/core/settings/keymap.json

```
@@ -80,6 +80,11 @@
     "key": "alt+h",
     "description": "Toggle selected region visibility"
   },
+  "region:visibility-all": {
+    "key": "ctrl+h",
+    "mac": "command+h",
```

```suggestion
    "mac": "ctrl+h",
```
`⌘+H` is a Mac global hotkey to hide a window, so let's use `ctrl+h` on every platform and explicitly state this in here.

---

good catch